### PR TITLE
Updating kustomize to the current latest version 5.3.0

### PR DESCRIPTION
**Pull request by ravisinghkr**
As part of pay.UK, we need to run `kustomize edit set configmap` which was added in the latest build.

### CODE REVIEW

**Review by ElvenSpellmaker — APPROVED** [review]

File: image_definitions/base/Dockerfile [hunk 1]
```diff
@@ -52,20 +52,20 @@ ENV TZ="Europe/London"
...
+    tar \
+    python3 \
+    python3-dev \
+    py3-pip \
+    build-base \
+    linux-headers \
+    docker-cli \
+    sudo \
+    shadow \
+    gnupg \
```
Comment (ElvenSpellmaker) [inline image_definitions/base/Dockerfile:68]: Wait one sec, why has this been brought back?

Comment (ElvenSpellmaker) [reply image_definitions/base/Dockerfile:68]: Can we put the indentation back please? 😁

**Review by ElvenSpellmaker — CHANGES_REQUESTED** [review]

File: image_definitions/base/Dockerfile (hunk 1 above)
Comment (ravisinghkr) [reply image_definitions/base/Dockerfile:68]: yeah it auto formatting by vs code, let me try to put it back :)

Comment (ElvenSpellmaker) [reply image_definitions/base/Dockerfile:68]: No worries, I wonder why it tries to do that =S

Comment (ravisinghkr) [reply image_definitions/base/Dockerfile]: done

**Review by ElvenSpellmaker — APPROVED** [review]

**Pull request merged**